molecule/default/tests/glance_helpers.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+import requests
+import json
+import re
+from cStringIO import StringIO
+
+
+""" Glance helpers
+"""
+
+
+def get_scoped_token(openrc):
+    """Getting project-scoped token
+    Project-scoped tokens are the bread and butter of OpenStack. They express your authorization to operate in a
+    specific tenancy of the cloud and are useful to authenticate yourself when working with most other services.
+    They contain a service catalog, a set of roles, and details of the project upon which you have authorization.
+
+    A project-scoped token is similar to the below string:
+    gBBBsfawerasaserse9"mLQilrmwtA-dfdfdfdfsasetrewfazsThelqeIE7nba3wgyxgrsawefsafawDuuxFMnY7E5hud_UT9bmI |
+    KY39DRIVbv85sXmfddzbFma0UUQtA0Z7oDsRUKVz1zKp-zHVxfdsfasebsdgasegzas"
+
+    Args:
+          openrc (str): string variable containing `openrc` authorization and authentication file
+
+    Return:
+          project-scoped token (str): `X-Subject-Token` in the requested headers.
```

Comment:
I know this is going to seem pedantic, but for `return` docstring statements you don't specify a name, but instead specify only the data type:

```python
str: `X-Subject-Token` in the requested headers.
```